File: server-scala-play-server/app/model/LoadResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for LoadResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class LoadResponse(
  id: Option[String],
  name: Option[String],
  size: Option[Long],
  loadingTime: Option[Long],
  separatePayloads: Option[Boolean],
  pickup: Option[StopResponse],
  dropoff: Option[StopResponse]
)

object LoadResponse {
  implicit lazy val loadResponseJsonFormat: Format[LoadResponse] = Json.format[LoadResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/LoadResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class LoadResponse 
{
    public string Id { get; set; }
    public string Name { get; set; }
    public long Size { get; set; }
    public long LoadingTime { get; set; }
    public bool SeparatePayloads { get; set; }
    public StopResponse Pickup { get; set; }
    public StopResponse Dropoff { get; set; }
}


